meshroom/ui/qml/Application.qml

```diff
@@ -20,6 +20,23 @@ Page {
     property alias unsavedDialog: unsavedDialog
     property alias workspaceView: workspaceView
 
+    property var scenefile: _reconstruction ? _reconstruction.graph.filepath : "";
+
+    // Signal to denote the current meshroom project has been changed
+    signal currentProjectChanged();
+
+    onScenefileChanged: {
+        // Check if we're not currently saving and emit the currentProjectChanged signal
+        if (! _reconstruction.graph.isSaving) {
+            currentProjectChanged();
+        }
```

Comment:
Couldn't we do
```suggestion
        if (! _reconstruction.graph.isSaving) {
            nodeEditor.refresh();
        }
```
?

That would prevent us from having one more signal/slot for something that can be done in a single operation.

Reply:
`currentProjectChanged` could thus be completely removed.

Reply:
The intention was to have a callback and then use that callback for this and other purposes in the future. I do agree with you since we don't have anything right now, let's use that itself and we can review this later when it will be needed.